makiflow/layers/trainable_layers.py

```diff
@@ -269,7 +282,7 @@ class BiasLayer(SimpleForwardLayer):
     D = 'D'
     TRAINABLE = 'trainable'
 
-    def __init__(self, D, name, trainable=True, b=None):
+    def __init__(self, D, name, trainable=True, regularize_bias=True, b=None):
         """
         BiasLayer adds a bias vector of dimension D to a tensor.
```

Comment:
Add commentary on why there is `trainable` argument.
If trainable set to False, then the bias is meant to be a constant. It is done this way to prevent the bias from accidential training if the user simply forgot to make this layer untrainable after creating a model.